Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarWebServiceTest.cs

```diff
@@ -915,23 +919,20 @@ public void TryDownloadEmbeddedFile_RequestedFileDoesNotExist()
         [TestMethod]
         public async Task GetProperties_Old_Forbidden()
         {
-            const string serverUrl = "http://localhost";
-            const string projectKey = "my-project";
-
             var responseMock = new Mock<HttpWebResponse>();
             responseMock.SetupGet(x => x.StatusCode).Returns(HttpStatusCode.Forbidden);
 
             var downloaderMock = new Mock<IDownloader>();
             downloaderMock
-                .Setup(x => x.Download(new Uri($"{serverUrl}/api/server/version"), false))
+                .Setup(x => x.Download(new Uri($"{ServerUrl}/api/server/version"), false))
                 .Returns(Task.FromResult("1.2.3.4"));
             downloaderMock
-                .Setup(x => x.Download(new Uri($"{serverUrl}/api/properties?resource={projectKey}"), true))
+                .Setup(x => x.Download(new Uri($"{ServerUrl}/api/properties?resource={ProjectKey}"), true))
                 .Throws(new HttpRequestException("Forbidden"));
 
-            var service = new SonarWebService(downloaderMock.Object, serverUrl, logger);
+            var service = new SonarWebService(downloaderMock.Object, ServerUrl, logger);
```

Comment:
This file needs a big cleanup (too many empty lines, repetitiions, comments). Not in this PR.

Reply:
I agree with both